src/screens/Topics/index.js

```diff
@@ -120,7 +123,7 @@ const Topics = () => {
           </View>
 
           {topics?.length > 0 && (
-            <ScrollView showsVerticalScrollIndicator={false} style={styles.scrollViewStyle}>
+            <ScrollView showsVerticalScrollIndicator={false} style={styles.scrollViewStyle(bottom)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Passing `bottom` to `scrollViewStyle` is a good practice as it allows dynamic calculation of marginBottom based on the safe area insets. This improves the visual consistency of the ScrollView component across different devices.
```diff
- <ScrollView showsVerticalScrollIndicator={false} style={styles.scrollViewStyle}>
+ <ScrollView showsVerticalScrollIndicator={false} style={styles.scrollViewStyle(bottom)}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->